apps/microsoft-teams/frontend/src/components/config/NotificationEditMode/NotificationEditMode.tsx

```diff
@@ -60,24 +61,22 @@ const NotificationEditMode = (props: Props) => {
         contentTypeId: notification.contentTypeId,
       };
 
-      const { response } = await sdk.cma.appActionCall.createWithResponse(
+      const { result, error } = await sdk.cma.appActionCall.createWithResult(
         {
           appActionId: 'msteamsSendTestMessage',
           environmentId: sdk.ids.environment,
           spaceId: sdk.ids.space,
           appDefinitionId: sdk.ids.app!,
-          userId: sdk.ids.user,
         },
         {
           parameters,
         }
       );
-      const body = JSON.parse(response.body);
 
-      if (body.ok) {
+      if (result && isSendTestMessageResult(result) && result.ok) {
```

Comment:
result.ok should not be needed, since presumably the parser isSendTestMessageResult should do that check already.